components/todo-item.js

```diff
@@ -1,11 +1,63 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem({ id, content, date, todoRemove }) {
+  return (
+    <Wrapper>
+      <TodoItemBox>
+        <Todo>{content}</Todo>
+        <Day>{date}
+        <ClearBtn onClick={(() => {todoRemove(id)})}>완료</ClearBtn>
+        </Day>
+      </TodoItemBox>
+    </Wrapper>
+    );
 }
 
 const Wrapper = styled.div`
-  font-size: 18px;
-  border: solid 1px;
+  display: flex;
+`;
+
+const TodoItemBox = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 3rem;
+  padding: 1rem 2rem;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  border-image: initial;
+`;
+
+const Todo= styled.p`
+  font-size: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const Day= styled.p`
+  font-size: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 0px;
+  margin: 0px;
+`;
```

Comment:
```suggestion
const Date= styled.p`
  font-size: 1.5rem;
  display: flex;
  flex-direction: column;
  margin: 0px;
`;
```
날짜니까 네이밍은 ```Date```가 적당할 것 같아용